eng/pipelines/extra-platforms/runtime-extra-platforms-android.yml

```diff
@@ -114,3 +114,45 @@ jobs:
           parameters:
             creator: dotnet-bot
             testRunNamePrefixSuffix: CoreCLR_$(_BuildConfig)
+
+#
+# Android arm64 devices and x64 emulators
+# Build the whole product using CoreCLR and run functional tests
+#
+- template: /eng/pipelines/common/platform-matrix.yml
```

Comment:
Consider adding to eng/pipelines/runtime-androidemulator.yml and runtime-android.yml as well. These pipelines are for when you only want to run against android and not the entire extra-platforms kitchen sink.

Reply:
Added to runtime-extra-platforms-androidemulator.yml. Looks like runtime-androidemulator.yml and runtime-android.yml pull in runtime-extra-platforms-androidemulator.yml and runtime-extra-platforms-android.yml, respectively.

Reply:
Yep, and you can also trigger them via `/azp run`